doc/sphinx/language/core/variants.rst

```diff
@@ -60,7 +60,7 @@ Definition by cases: match
 .. insertprodn term_match pattern0
 
 .. prodn::
-   term_match ::= match {+, @case_item } {? return @term100 } with {? %| } {*| @eqn } end
+   term_match ::= match {+, @case_item } {? return @term100 } with {? {| {? %| } {*| @eqn } | %| {+| @branch } | {+| @branch } } } end
```

Comment:
Ltac2 extends the `match` syntax on terms!?

Reply:
Well, yes.  There are additions to `branches` that refer to many other Ltac2-defined nonterminals.

![image](https://user-images.githubusercontent.com/1253341/84417539-a187da80-abca-11ea-8b17-81273f4bcfad.png)

Reply:
OK, I see the problem: Ltac2 defines its own `branches` and `branch` non-terminals, it doesn't extend the existing ones.